src/lsdb/io/to_hats.py

```diff
@@ -90,6 +92,10 @@ def to_hats(
             exist.
         histogram_order (int): The default order for the count histogram. Defaults to 8.
         overwrite (bool): If True existing catalog is overwritten
+        create_thumbnail (bool): If True, create a data thumbnail of the catalog for previewing purposes.
+            Defaults to False.
+        thumbnail_threshold (int): The number of points per pixel as specified at import time.
```

Comment:
```suggestion
        thumbnail_threshold (int): The maximum number of rows in the data thumbnail,
            which is otherwise one row/partition.
```